packages/nimble-components/src/combobox/index.ts

```diff
@@ -153,9 +153,7 @@ export class Combobox
         let updatedValue = next;
 
         if (this.$fastController.isConnected && this.options) {
-            const selectedIndex = this.options.findIndex(
-                el => el.text.toLowerCase() === next.toLowerCase()
-            );
```

Comment:
Looking at this implementation (recognizing that this is just copied from FAST), I'm concerned that this isn't really the behavior we want. While being case-insensitive makes sense for filtering, actually setting the `selectedIndex` to the first option found in a case-insensitive manner seems wrong. In fact, before your changes, I believe the `inputHandler` did things as I would expect, it would set the `selectedIndex` to the value considering case (but also would then set the `value` at the end, which could actually have resulted in buggy behavior).

Reply:
So is the expectation that the control will only coerce the value typed in (to match the casing of a dropdown option) when the `autocomplete` mode is `inline` or `both`? I.e. if I type in "s", it will autocomplete to "Sue". Otherwise, if I just commit the value "sue" (either by keyboard interaction or programatically), the control value stays "sue" and the `selectedIndex` is -1. Is that right?

Reply:
I've updated the code (and one test) to work according to that understanding.

Reply:
So, to be clear, the case of concern is when there is both the option "Sue" and "sue", and specifically when "Sue" is after "sue" in order. In this case, if a user types "Sue" I would expect the filtered list to show both "sue" and "Sue", but I would expect "Sue" to be highlighted (though this part is admittedly subjective). What we _don't_ want (and I don't think your changes allow this, but prior to them I believe this could occur) is that while typing "Sue", "Sue" would be highlighted as expected, but when committing the value, you would find that "sue" was selected instead.

Reply:
Yes, I believe that is right. Admittedly, the scenario I thought I was concerned with can't really happen. I figured if you had both "Sue" and "sue", and typed "S", you would want "Sue" to the one that got selected. However, it appears that you actually can't even type in a capital letter if there is an option that is first matched that doesn't start with a capital letter. Personally, I think this is kinda crappy, but I'm not interested in holding this PR up for existing behavior that is admittedly a bit subjective.